frontends/web/src/locales/en/app.json

```diff
@@ -1570,6 +1570,7 @@
       "successfullySigned": "Request succesfully signed",
       "walletConnectRequest": "WalletConnect request"
     },
+    "useNewUri": "Please use a new URI",
```

Comment:
I think it would be nice to have a more descriptive error, something like "This URI has been already used to attempt a connection. Please use a new URI"
Not a strong opinion, though :)